config/Common.xcconfig

```diff
@@ -1,9 +1,29 @@
+// Shared
+
```

Comment:
Nitpick: It's sort of implied that these are shared settings given we're in "common".

```suggestion
```

But okay to leave it if you want homogeneous separation headings.